apps/hashdotdev/src/pages/roadmap/technology-tree.tsx

```diff
@@ -205,7 +205,13 @@ export const TechnologyTree: FunctionComponent = () => {
           completion status.
         </Typography>
       </Container>
-      <Container>
+      <Container
+        sx={{
+          maxWidth: {
+            lg: 1500,
```

Comment:
@nonparibus as you suggested in Figma I've made the technology tree wider than the paragraph elements (up to 1500px). Let me know if you'd like me to adjust it to something else.

<img width="2220" alt="image" src="https://github.com/hashintel/hash/assets/42802102/03f51c62-2e23-4ee7-9f50-7f011bf480ce">

Reply:
On the latest deployment the right-hand filters look different for me than in the screenshot above. Any idea why? 
<img width="1489" alt="Capture 2023-07-31 at 19 36 27@2x" src="https://github.com/hashintel/hash/assets/6226576/aa8fad9a-5ddf-48bd-beb1-72d86760993c">

Reply:
Because at that screen width the filter would overflow the page if we were to stick it on the right of the graph. If you were to make your screen wider it should appear on the right when there is enough space for it.